### ISSUE TITLE
Running seed file multiple times results results in an error

### ISSUE THREAD

**Issue by phantompatrol** [umts/probable-engine#123]
There is a bug in the seeds file caused by the script attempting to insert a duplicate vehicle:
```Ruby
ActiveRecord::RecordNotUnique: Mysql2::Error: Duplicate entry 'f51baded-5e26-4727-bd52-fa3ddccd0e18' for key 'index_items_on_uuid': INSERT INTO `items` (`name`, `item_type_id`, `uuid`, `created_at`, `updated_at`) VALUES ('Kanto', '1', 'f51baded-5e26-4727-bd52-fa3ddccd0e18', '2016-07-29 13:55:27', '2016-07-29 13:55:27')
/Users/logan/projects/probable-engine/db/seeds.rb:68:in `block (2 levels) in <top (required)>'
/Users/logan/projects/probable-engine/db/seeds.rb:67:in `each'
/Users/logan/projects/probable-engine/db/seeds.rb:67:in `block in <top (required)>'
/Users/logan/projects/probable-engine/db/seeds.rb:66:in `<top (required)>'
/Users/logan/.rbenv/versions/2.3.1/bin/bundle:23:in `load'
/Users/logan/.rbenv/versions/2.3.1/bin/bundle:23:in `<main>'
Mysql2::Error: Duplicate entry 'f51baded-5e26-4727-bd52-fa3ddccd0e18' for key 'index_items_on_uuid'
/Users/logan/projects/probable-engine/db/seeds.rb:68:in `block (2 levels) in <top (required)>'
/Users/logan/projects/probable-engine/db/seeds.rb:67:in `each'
/Users/logan/projects/probable-engine/db/seeds.rb:67:in `block in <top (required)>'
/Users/logan/projects/probable-engine/db/seeds.rb:66:in `<top (required)>'
/Users/logan/.rbenv/versions/2.3.1/bin/bundle:23:in `load'
/Users/logan/.rbenv/versions/2.3.1/bin/bundle:23:in `<main>'
Tasks: TOP => db:seed
(See full trace by running task with --trace)
```

Please refactor the seeds file to check to see if the cart exists, before inserting. Explanation that is not meant to sound `preachy`, but totally does: As a general rule, the seeds should be [idempotent](https://en.wikipedia.org/wiki/Idempotence). That is, no matter when and how often they run, they should always ensure the app contains (at a minimum) the objects and relationships outlined in the seed file(s).

**Comment by phantompatrol** [comment]
@Cobchise, may I put this into your queue?

**Comment by Cobchise** [comment]
@phantompatrol What's it worth to you?? 💰💰  .  Yeah I can work on this 😃  Had intended to work on it after you mentioned it to me before

**Comment by phantompatrol** [comment]
@Cobchise Would your new base rate work 👅 ?

Yeah, I knew this was on your radar, but I just getting a base line on the whole app so that nothing gets missed. I often make issues fully intending on resolving them myself, just so I have a roadmap. It is a good (if noisy) habit.

**Comment by phantompatrol** [comment]
RE: https://github.com/umts/probable-engine/issues/123#event-739271522 WHOOPS, my hand slipped on my trackpad and I accidentally assigned it...

**Comment by Cobchise** [comment]
@phantompatrol Done and done (I hope)

**Comment by phantompatrol** [comment]
Done indeed!

**Comment by phantompatrol** [comment]
FYI, if you put `Fixes #123` or `Resolves #123` in your commit message, it should auto close the issue!

https://help.github.com/articles/closing-issues-via-commit-messages/
